allowed-breaking-changes.txt

```diff
@@ -47,4 +47,12 @@ incompatible-argument:@aws-cdk/aws-ecs.Ec2TaskDefinition.addVolume
 incompatible-argument:@aws-cdk/aws-ecs.FargateTaskDefinition.<initializer>
 incompatible-argument:@aws-cdk/aws-ecs.FargateTaskDefinition.addVolume
 incompatible-argument:@aws-cdk/aws-ecs.TaskDefinition.<initializer>
-incompatible-argument:@aws-cdk/aws-ecs.TaskDefinition.addVolume
\ No newline at end of file
+incompatible-argument:@aws-cdk/aws-ecs.TaskDefinition.addVolume
+
+# The new ELBv2 context queries added these arms to the ContextQueryProperties
+# union:
+# | LoadBalancerContextQuery
+# | LoadBalancerListenerContextQuery
+# | SecurityGroupContextQuery
+# These additions changed the signature of loadAssemblyManifest()
+change-return-type:@aws-cdk/cloud-assembly-schema.Manifest.loadAssemblyManifest
```

Comment:
Changing line 58 to the following fixes the problem with this PR's build, introduced when merging in JSII 1.14 from master:

```
weakened:@aws-cdk/cloud-assembly-schema.MissingContext
````